code/modules/admin/verbs/buildmode.dm

```diff
@@ -293,7 +293,7 @@
 					var/turf/T = get_turf(object)
 					log_admin("Build Mode: [key_name(user)] modified [T] ([T.x],[T.y],[T.z]) to [objholder]")
 					T.ChangeTurf(objholder)
-				else
+				else if(objholder)
```

Comment:
Giff +BUILDMODE and I'll tell ;)

Seriously though, I assume it's due to clicking in buildmode with no object selected